internal/overlord/servstate/manager_test.go

```diff
@@ -167,7 +175,7 @@ func (s *S) SetUpTest(c *C) {
 
 	restore := servstate.FakeOkayWait(shortOkayWait)
 	s.AddCleanup(restore)
-	restore = servstate.FakeKillWait(shortKillWait, shortFailWait)
+	restore = servstate.FakeKillDelayWait(shortKillDelay, shortKillWaitDuration)
```

Comment:
That emphasizes one of the earlier points: the delta is "kill" vs. "fail", not "wait" vs. "delay". vs. "duration